### PR TITLE
Update base images in the dockerfiles

### PR DESCRIPTION
**Pull request by harshad16**
Update base images in the dockerfiles
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

### CODE REVIEW

**Review by Gregory-Pereira — COMMENTED** [review]
/lgtm
however the f35-py310 overlay didnt build tag `v0.33.0`

File: .thoth.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ runtime_environments:
       version: "35"
     python_version: "3.10"
     recommendation_type: latest
-    base_image: quay.io/thoth-station/s2i-thoth-f35-py310:v0.32.3
+    base_image: quay.io/thoth-station/s2i-thoth-f35-py310:v0.33.0
```
[the remark this answers is not in the document]
Comment (harshad16) [reply .thoth.yaml:37]: I have fixed it now.
thanks :+1:

File: overlays/f35-python310/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM quay.io/thoth-station/s2i-thoth-f35-py310:v0.32.3
+FROM quay.io/thoth-station/s2i-thoth-f35-py310:v0.33.0
```
[the remark this answers is not in the document]
Comment (harshad16) [reply overlays/f35-python310/Dockerfile:1]: I have fixed it now.
thanks +1

**Pull request merged**